ROADMAP.md

```diff
@@ -1,7 +1,64 @@
 # KF Serving 2019 Roadmap
-## Q2 2019
+## Q3 2019
+### v0.2 Integrate with the ML Ecosystem (ETA: August 15, 2019)
```

Comment:
And have examples for 'Cloud Providers':  e.g. getting it running on Azure, IBM Cloud etc

Reply:
Sure